### ISSUE TITLE
Getting spaces from Julich parcellation kills python kernel on refactor_v0.4a8

### ISSUE THREAD

**Issue by AhmetNSimsek** [FZJ-INM1-BDA/siibra-python#230]
The following code
```
import siibra
siibra.parcellations["julich"].spaces
```
caused kernel to crash:
"Canceled future for execute_request message before replies were done
The Kernel crashed while executing code in the the current cell or a previous cell."

**Comment by xgui3783** [comment]
can confirm

my stacktrace looks like

```
  File "/home/xgui3783/dev/projects/siibra-python/venv/lib/python3.8/site-packages/anytree/iterators/abstractiter.py", line 44, in __next__
    self.__iter = self.__init()
  File "/home/xgui3783/dev/projects/siibra-python/venv/lib/python3.8/site-packages/anytree/iterators/abstractiter.py", line 28, in __init
    children = [] if AbstractIter._abort_at_level(1, maxlevel) else AbstractIter._get_children([node], stop)
  File "/home/xgui3783/dev/projects/siibra-python/venv/lib/python3.8/site-packages/anytree/iterators/abstractiter.py", line 57, in _get_children
    return [child for child in children if not stop(child)]
  File "/home/xgui3783/dev/projects/siibra-python/venv/lib/python3.8/site-packages/anytree/iterators/abstractiter.py", line 57, in <listcomp>
    return [child for child in children if not stop(child)]
RecursionError: maximum recursion depth exceeded
```

**Comment by dickscheid** [comment]
Fixed in 9c7a60d  - this was only a problem in Parcellation.supported_spaces, which was an old implementation and could be removed to fallback to the parent class Region.supported_spaces that had a better implementation already.
